src/components/CList/CList.js

```diff
@@ -163,7 +163,6 @@ const getClientPagination = (config, setPagination) => {
     rowsPerPage: config.rowsPerPage,
     sortBy: sort(),
     descending: config.sort ? config.sort === 'desc' : null,
-    page: config.startPage,
```

Comment:
There is an issue on table element after this update. It's not setting data (you can test it locally). I believe you still need to set page here. Just set default **1** value on page.